graylog2-web-interface/src/routing/context/ScratchpadProvider.jsx

```diff
@@ -0,0 +1,27 @@
+
```

Comment:
Not sure if there's a linting rule for this, but I guess we can remove this empty line :)